NEMP/mods.json

```diff
@@ -4101,24 +4227,6 @@
             "regex" : "BiomeTweaker-(?P<mc>.+?) (?P<version>.+)"
         }
     },
-    "Caterpillar": {
-        "function": "CheckCurse",
-        "active": true,
-        "curse": {
-            "regex": "Caterpillar-(?P<version>.+)\\.jar",
-            "id": "237628",
-            "name": "caterpillar"
-        }
-    },
-    "UnLogicII": {
```

Comment:
It isn't, I just wanted to make sure it wasn't a mistake from the reorganization.